test/lib/analyzer/python-pip-analyzer.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { readdirSync, readFileSync } from "fs";
+import * as path from "path";
+import { pipFilesToScannedProjects } from "../../../lib/analyzer/applications";
+import { FilePathToContent } from "../../../lib/analyzer/applications/types";
+
+function prepareFiles(testPath: string): FilePathToContent {
+  const basePath = path.join(__dirname, `../../fixtures/python/${testPath}`);
+  const filePathToContent = {
```

Comment:
Non blocking:
There are nice helpers for loading fixtures in `test/util/index.ts` that handle the path, `toString` etc